platform/ios/ios.xcworkspace/xcshareddata/xcdebugger/Breakpoints_v2.xcbkptlist

```diff
@@ -11,14 +11,6 @@
             continueAfterRunningActions = "No"
             scope = "1"
             stopOnStyle = "0">
-            <Actions>
```

Comment:
Eh? Probably worth breaking out into a different PR if you don’t like the default (but disabled) breakpoint. 😉